ganttproject/src/net/sourceforge/ganttproject/gui/GanttDialogPerson.java

```diff
@@ -58,6 +58,7 @@
   private final StringOption myPhoneField = new DefaultStringOption("colPhone");
   private final StringOption myMailField = new DefaultStringOption("colMail");
   private final MoneyOption myStandardRateField = new DefaultMoneyOption("colStandardRate");
+  private final MoneyOption myTotalCostField = new DefaultMoneyOption("colTotalCost");
```

Comment:
Pls make it read-only. There is `setWritable` method in the implementation class.